### DISCUSSION TITLE
polymer capacitor

### DISCUSSION THREAD

**Discussion by class976** [jomjol/AI-on-the-edge-device#2621] [Q&A] [answered]
Hi. i am looking for polymer capacitor for external led
where can i buy this? and what size in voltage do you recomment for the esp32cam
when i search for polymer i get a tons of websites with all kinds of this capacitor.
greetings

**Comment by caco3** [comment, marked as answer]
Do you refer to the example in https://jomjol.github.io/AI-on-the-edge-device-docs/External-LED/#1-hardware-installation-of-the-led-stripe?

Just use a simple electrolyte capacitor. It must be rated for at least 5V (better 10V). The capacitance itself is not that critical, anything 100 ..1000 uF will do the job.
Make sure to  wire it up correctly (it has a plus and a minus pole)!

For testing you can also keep it aside, it is just there to support the power supply.

**Comment by class976** [comment]
Thank you very much. I got my self a capacitor 1000uf 16volt and it works fine.
